_includes/contact_form.html

```diff
@@ -402,6 +402,7 @@
       <option value="USAJOBS">
         {{ site.data.[page.lang].settings.contact_page.support_request_form.agencies.usajobs }}
       </option>
+      <option value="PBGC - MyPBA">PBGC - MyPBA</option>
```

Comment:
I know we typically go through i18n for this.... but my thinking is that since we never translate the names of these options anyways, maybe it's ok to just hardcode them like this

I'd love feedback on this, ex if we want to go through i18n for now and move them all to be hardcoded together

Reply:
We do translate a few, like "DOD Move.mil (Homes or Office)" -> "DOD Move.mil (Hogares u oficinas)". To me, the fact that most agency names don't translate is more incidental to the fact that we'd want these labels to be translated.

If the goal would be to reduce effort / duplication, would it be possible to have the locale data serve as the source of truth, and then generate these options by looping through said data?

Reply:
Yeah one thing the old i18n gem did give us was fallback behavior, falling back to `en` if there wasn't a locale translation... I have an idea for cleaning this up with a custom plugin

Reply:
and alphabetization is important too